### PR TITLE
storage: Robustify btrfs polling

### PR DESCRIPTION
**Pull request by mvollmer**
It is only a warning when one of the "btrfs" invocations terminates unsuccessfully, and the loop continues with the next volume.

Some failures are expected when polling happens concurrently with removing subvolumes or volumes.

### CODE REVIEW

File: pkg/storaged/client.js [hunk 1]
```diff
@@ -258,50 +257,47 @@ export async function btrfs_poll() {
...
             } catch (err) {
-                console.error(`unable to obtain subvolumes for mount point ${mount_point}`, err);
-                return;
-            }
-            const subvols = [{ pathname: "/", id: 5 }];
-            for (const line of output.split("\n")) {
-                const m = line.match(/ID (\d+).*parent (\d+).*path (<FS_TREE>\/)?(.*)/);
-                if (m)
-                    subvols.push({ pathname: m[4], id: Number(m[1]) });
+                console.warn(`unable to obtain subvolumes for mount point ${mount_point}`, err);
```
Comment (cockpituous) [inline pkg/storaged/client.js:269]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19853-20240118-095029-5e050238-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/client.js [hunk 2]
```diff
@@ -258,50 +257,47 @@ export async function btrfs_poll() {
...
             // https://github.com/storaged-project/udisks/commit/b6966b7076cd837f9d307eef64beedf01bc863ae
             try {
-                output = await cockpit.spawn(["btrfs", "subvolume", "get-default", mount_point], { superuser: true, err: "message" });
+                const output = await cockpit.spawn(["btrfs", "subvolume", "get-default", mount_point], { superuser: true, err: "message" });
                 const id_match = output.match(/ID (\d+).*/);
                 if (id_match)
                     btrfs_default_subvol[uuid] = Number(id_match[1]);
             } catch (err) {
-                console.error(`unable to obtain default subvolume for mount point ${mount_point}`, err);
+                console.warn(`unable to obtain default subvolume for mount point ${mount_point}`, err);
```
Comment (cockpituous) [inline pkg/storaged/client.js:281]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19853-20240118-095029-5e050238-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/storaged/client.js [hunk 3]
```diff
@@ -258,50 +257,47 @@ export async function btrfs_poll() {
...
+                for (const line of usage_output.split("\n")) {
+                    const match = usage_regex.exec(line);
+                    if (match) {
+                        const { used, device } = match.groups;
+                        usages[device] = used;
+                    }
                 }
+                uuids_usage[uuid] = usages;
+            } catch (err) {
+                console.warn(`btrfs filesystem show ${uuid}`, err);
```
Comment (cockpituous) [inline pkg/storaged/client.js:299]: These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19853-20240118-095029-5e050238-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by jelly — APPROVED** [review]
If the tests agree!

**Pull request merged**